Library/Formula/flif.rb

```diff
@@ -0,0 +1,24 @@
+class Flif < Formula
+  desc "Free Lossless Image Format"
+  homepage "http://flif.info"
+  url "https://github.com/FLIF-hub/FLIF/archive/v0.1-alpha.tar.gz"
+  version "0.1-alpha"
```

Comment:
@mikemcquaid OK, I will make a PR to `homebrew-devel-only` after we finish with this review.